pip_review/__main__.py

```diff
@@ -244,55 +136,59 @@ def confirm(question):
     return answer == 'y'
 
 
+def get_outdated_packages(local=False, pre_release=False, user=False):
+    command = ['pip', 'list', '--outdated', '--disable-pip-version-check']
```

Comment:
I would really prefer for the build to succeed before merging. But if you get fed up with it, I will give it a try myself.